### DISCUSSION TITLE
403 Forbidden You don't have permission to access this resource

### DISCUSSION THREAD

**Discussion by AnythingLinux** [crater-invoice-inc/crater#1292] [General]
After setup 403 Forbidden You don't have permission to access this resource

**Comment by Javiermona** [comment]
Hello, I want to know how I can set a negative tax. In Spain, there is a 15% negative withholding. The app does not allow setting negative taxes. What I did was to set the value to negative in the database and changed 15 to -15, and it accepted it. I also modified a field in the invoice table to respect the negative value. Now what happens is that the reports show an error due to this change I made. I don't know Laravel, and it is very difficult for me to find a way to develop it. Does anyone have an idea on how to do it?

**Comment by Javiermona** [comment]
![negative ](https://github.com/crater-invoice/crater/assets/31548891/432336fa-fdcf-4ed7-bf8c-c12ed71f0863)
